cypress/integration/2-page-component-tests/checkboxes.cypress.js

```diff
@@ -0,0 +1,49 @@
+const path = require('path')
+
+const { waitForApplication } = require('../utils')
+
+const templatesView = path.join(Cypress.config('fixturesFolder'), 'views', 'checkbox-test.html')
+const appView = path.join(Cypress.env('projectFolder'), 'app', 'views', 'checkbox-test.html')
+
+const pagePath = '/checkbox-test'
+
+describe('checkbox tests', () => {
+  before(() => {
+    waitForApplication()
+
+    // load test view
+    cy.task('deleteFile', { filename: appView })
+    cy.task('log', `Copy ${templatesView} to ${appView}`)
+    cy.task('copyFile', { source: templatesView, target: appView })
+    cy.task('log', 'The checkbox-test page should be displayed')
+    cy.visit(pagePath)
+    cy.get('h1')
+      .should('contains.text', 'Checkbox tests')
+  })
+
+  after(() => {
+    waitForApplication()
+    cy.task('deleteFile', { filename: appView })
+  })
+
+  it('request should include _unchecked option only', () => {
+    cy.intercept('POST', pagePath).as('submitPage')
+    cy.get('button[data-module="govuk-button"]')
+      .should('contains.text', 'Continue')
+      .click()
+    cy.wait('@submitPage').its('request.body')
+      .then((body) => cy.expect(decodeURIComponent(body))
+        .equal('vehicle1[vehicle-features]=_unchecked'))
+  })
+
+  it('request should include GPS option', () => {
```

Comment:
I'm not sure this test title is very descriptive, because to me it doesn't really explain why the request should include the GPS option, or why we should care...